DiscordCode/BackEnd/routes/AccountSettings.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const user = require('./models/User');
+
+mongoose.connect('mongodb://localhost:27017/colligo');
+const db = mongoose.connection;
+
+const router = express.Router();
+
+router.post('/', async (req, res) => {
```

Comment:
Should be sending json data back so frontend can display this messages to the user.